pyproject.toml

```diff
@@ -95,13 +95,90 @@ dev = [
     "types-psutil==5.8.23",
     "types_requests==2.31.0.20240406",
     "types-protobuf==5.26.0.20240422",
+    "deptry==0.14"
 ]
 build = [
     "pyinstaller==6.6.0",
     "setuptools==69.5.1",
     "build==1.2.1"
 ]
 
+[tool.deptry]
+extend_exclude = [
+    "doc",
+    "scripts",
+    "sigs",
+    "tests"
+]
+
+# X is marked as first party, to inform deptry that it is local
+known_first_party = [
+    'PyQt5',
+    'backports',
+    'binaryninja',
+    'flirt',
+    'ghidra',
+    'ida_bytes',
+    'ida_entry',
+    'ida_funcs',
+    'ida_kernwin',
+    'ida_loader',
+    'ida_nalt',
+    'ida_segment',
+    'idaapi',
+    'idautils',
+    'idc',
+    'java',
+    'netnode'
+]
+
+# X defined as a dependency but not used in the codebase
+[tool.deptry.per_rule_ignores]
+DEP002 = [
+    "deptry",
```

Comment:
let's sort these a-z?

Reply:
Sorry, I was using a mix of single and double quotes which resulted incorrect order when sorting in Vim. I used the standard `"` instead.